pkg/files/manager.go

```diff
@@ -98,9 +96,41 @@ func (fm *FileManagerImpl) DeleteFile(ctx context.Context, fs afero.Fs, path str
 	return fs.Remove(path)
 }
 
-func (fm *FileManagerImpl) ListFiles(ctx context.Context, fs afero.Fs, showHidden bool, filter PatternFilter) ([]*model.File, error) {
+type ListFilesOptions struct {
```

Comment:
np: Maybe move it to a different file?

Reply:
I think we should split manager in multiple files, would be strange to move out just options for `ListFiles()`.